proteus/MeshAdaptPUMI/createAnalyticGeometry.cpp

```diff
@@ -668,7 +668,7 @@ gmi_model* MeshAdaptPUMIDrvr::createSphereInBox(double* boxDim,double*sphereCent
   }
   m->end(it);
 
-  gradeMesh();
+  gradeMesh(1.5);
```

Comment:
We should eventually push this up to a parameter definition.

Reply:
Yes, this will likely happen during the broader refactoring of MeshAdaptPUMI. I'll add an issue for this so it won't be forgotten (see #1081)